packages/docs-components/src/CodeSnippet/CodeSnippetLabel.tsx

```diff
@@ -28,14 +43,14 @@ const CodeSnippetLabel: React.FunctionComponent<CopySnippetLabelProps> = props =
           lineHeight: 1,
           padding: '0.2rem 0.35rem',
           position: 'absolute',
-          right: '1rem',
-          top: '1rem',
+          right: '0.8rem',
+          top: '0.8rem',
           zIndex: 100,
         }}
         title={copyable ? 'Copy' : undefined}
       >
         <div>{label || mode}</div>
-        {copyable && <div style={{ marginLeft: '5px' }}>{active ? '✔ ' : '⎘'}</div>}
+        {copyable && <div style={{ marginLeft: '5px' }}>{active ? checkIcon : copyIcon}</div>}
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/60091217-6172e200-9744-11e9-97c3-c84d85480036.png)

Current icon looks ugly and there are no good replacement, so I decided to use SVGs.
Also it's not enough contrast, so change `color` to `#ccc` from PrismJS theme.


### After

![codesnippet-icons](https://user-images.githubusercontent.com/14183168/60091392-c595a600-9744-11e9-877c-d2a214f36403.gif)